client/app/scripts/components/node-details/node-details-table-row.js

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
 import classNames from 'classnames';
+import groupBy from 'lodash/groupBy';
+import forEach from 'lodash/forEach';
+import mapValues from 'lodash/mapValues';
```

Comment:
Now that we are using Webpack 2 in Scope, tree shaking should already do this for you if you go with `import { groupBy, forEach, mapValues } from 'lodash';`.